webcompat/static/js/lib/issues.js

```diff
@@ -489,6 +490,23 @@ issues.MainView = Backbone.View.extend({
 
   handleKeyShortcuts: function() {
     Mousetrap.bind("mod+enter", _.bind(this.addNewComment, this));
+  },
+
+  // Auto expanding feature for comments
```

Comment:
We could make a global function, but currently there isn't a "global.js" file -- ideally we're moving to a world of modules where we just pull in what we need. I think for now having it duped for now is fine, but we should add comments to both methods.